app/assets/javascripts/provider/admin/apiconfig/services/service_discovery.js

```diff
@@ -6,7 +6,7 @@ document.addEventListener("DOMContentLoaded", function(e) {
 
   function clearSelectOptions(select) {
     const options = select.getElementsByTagName('option');
-    for (let i = options.length -1; i > 0; i--) {
+    for (var i = options.length -1; i > 0; i--) {
```

Comment:
I don't understand the reason of this change 🤔 
We are [officially currently supporting](https://access.redhat.com/articles/2798521) these browsers:
![image](https://user-images.githubusercontent.com/11318903/46676229-59880780-cbe0-11e8-8ff8-c459d99d8894.png)
And all them support `let`:
https://caniuse.com/#feat=let

Reply:
Because tests are failing. Googling, I found Capybara-webkit/Karma/ doesn't support ES6
To use ES6 we should add this files into webpack pipeline.
But that will be done later, is it right @didierofrivia  ?

Reply:
aaah cool 👍 Thanks for the explanation 😄

Reply:
Strange. Our docker image is using firefox 52
Can you point me where it fails @damianpm

Reply:
@didierofrivia I remembered you solve some tests failing like this using selenium tag. What do you think is the best way? Not using JS or using selenium (i.e real browser) ?

Reply:
I'd like to be able to use all ES6 features, but for that maybe is better to create a new tasks, to transpile with babel/webpack.